doc/openapi.yaml

```diff
@@ -24,6 +24,23 @@ paths:
                         application/json:
                             schema:
                                 $ref: '#/components/schemas/GetProjectsResponse'
+    /api/1/task_data_add:
```

Comment:
Why not having something like ?:
- /api/1/task_data/add
- /api/1/task_data/commit
- /api/1/task_data/get_last_updated]
- /api/1/projects/get